### ISSUE TITLE
[Feature] Sync for SwitchyOmega

### ISSUE THREAD

**Issue by GEkuL** [FelisCatus/SwitchyOmega#2293]
### SwitchyOmega version / SwitchyOmega 版本
Version 2.5.20

### Browser version & OS version / 浏览器名称、版本及操作系统版本
Version 99.0 (64-bit)

### Problem description / 问题描述
Feature request

(Please provide as much detail as possible. We recommend the following format.)
(请尽可能多提供一些细节。我们推荐使用下面的格式。)

Hi there, 
It would be nice to have an external server for the ability to sync on different devices.
It could be realized with current function of Import/Export tab _Restore from online_ . Thus, a user should just set a link for file with the permissions read and write. It can be any cloud storage provider.
And after providing the link user should tick a point-box turning on the auto-sync.

In other words I'm suggesting to upgrade the _Restore from online_ function, making it automatic (with on demand manual sync) and making it working in both Read and Write modes.

This will improve a lot user experience in different browsers, keeping privacy on a high level.
